src/styles/_reset.scss

```
@@ -0,0 +1,16 @@
+*, *::before, *::after {
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body, html {
+  height: 100%;
```

напоминаю, что так делать не стоит)
https://github.com/week-password/wlss-frontend/issues/79

---

Правильно ли я понимаю, что, исходя из описания твоего тикета, в моём случае этого достаточно?
```suggestion
  min-height: 100%;
```

---

да, всё верно